### ISSUE TITLE
Occupation associated with frequent hand trauma

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10699]
**Preferred term label:**
Occupation associated with frequent hand trauma

**Synonyms**

**Definition (free text, please give PubMed ID)**
Occupations in which use of hands is frequent such as construction, food preparation and serving related occupations, and transportation and material moving occupations

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Extensor tendon laceration, hand/wrist

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Wouldn't it be clearer just to report a history of hand trauma?

**Comment by MickeySegal** [comment]
After submitting this we added 19 other diagnoses with this finding, many of which reflect chronic damage, e.g., ganglion cyst, pronator syndrome, ulnar artery thrombosis and digital artery aneurysm.  So it seems that there is value in specifying the occupational exposure.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001195
- label: Occupational exposure to frequent hand trauma
- parents: [Occupational exposure history (HP:0035010)](https://hpo.jax.org/app/browse/term/HP:0035010)
- pmid: 
- def: A history of exposure to repeated hand trauma as observed with occupations in which use of hands is frequent such as construction, food preparation and serving related occupations, and transportation and material moving occupations.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10699
- synonyms:
